pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -484,16 +502,51 @@ func (c *CertRotationController) RunOnce() error {
 	return utilerrors.NewAggregate(errlist)
 }
 
-func (c *CertRotationController) Run(ctx context.Context, workers int) {
+func (c *CertRotationController) Run(ctx context.Context) {
+	defer utilruntime.HandleCrash()
+
+	var wg sync.WaitGroup
 	klog.Infof("Starting CertRotation")
-	defer klog.Infof("Shutting down CertRotation")
+	defer func() {
```

Comment:
this change looks distinct.  What breaks if we don't have it and more importantly, get it merged separately.  It's buried wher eit is and this construct is clearly new.